unit_tests/child_chain/test_child_chain.py

```diff
@@ -4,21 +4,37 @@
 from mockito import any, mock, verify, when
 
 from plasma_cash.child_chain.block import Block
+from plasma_cash.child_chain.transaction import Transaction
 from plasma_cash.child_chain.child_chain import ChildChain
-from plasma_cash.child_chain.exceptions import InvalidBlockSignatureException
+from plasma_cash.child_chain.exceptions import (InvalidBlockSignatureException,
+                                                InvalidTxSignatureException,
+                                                PreviousTxNotFoundException,
+                                                TxAlreadySpentException,
+                                                TxAmountMismatchException)
 
 
 class TestChildChain(object):
     DUMMY_AUTHORITY = b"\x14\x7f\x08\x1b\x1a6\xa8\r\xf0Y\x15(ND'\xc1\xf6\xdd\x98\x84"
     DUMMY_SIG = '01' * 65  # sig for DUMMY_AUTHORITY
+    DUMMY_TX_NEW_OWNER = b'\xfd\x02\xec\xeeby~u\xd8k\xcf\xf1d.\xb0\x84J\xfb(\xc7'
     ROOT_CHAIN = mock()
 
     @pytest.fixture(scope='function')
     def child_chain(self):
+        DUMMY_TX_OWNER = b'\x8cT\xa4\xa0\x17\x9f$\x80\x1fI\xf92-\xab<\x87\xeb\x19L\x9b'
+
         deposit_filter = mock()
         when(self.ROOT_CHAIN).on('Deposit').thenReturn(deposit_filter)
-        return ChildChain(authority=self.DUMMY_AUTHORITY,
-                          root_chain=self.ROOT_CHAIN)
+        _child_chain = ChildChain(authority=self.DUMMY_AUTHORITY,
+                                  root_chain=self.ROOT_CHAIN)
+
+        # create a dummy transaction
+        tx = Transaction(0, 1, 10, DUMMY_TX_OWNER)
```

Comment:
might be better to write in Transaction(pre_block=xxx, uid=xxx, ,...) fashion for better readability